tests/unittests/utils/test_depth_anything.py

```
@@ -81,8 +87,11 @@ def test_use_ray_pose_explicit(self):
 class TestDepthAnythingV3OutputProcessor:
     """Test DepthAnythingV3OutputProcessor."""
 
-    def _make_processor(self) -> "DepthAnythingV3OutputProcessor":
-        from fiftyone.utils.depth_anything import DepthAnythingV3OutputProcessor
+    def _make_processor(self) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_make_processor`

(ANN401)